tailwind.config.ts

```diff
@@ -7,6 +7,8 @@ module.exports = {
         devstation: {
           "green-100": "#114C03",
           "green-200": "#498A39",
+          "project-card-bg-green": "#3B7227B2",
+          "project-card-border-green": "#529546",
```

Comment:
call em green-300 and green-400 and make sure you update the usage of them within projectcard.tsx